rules/REQUEST-903.9002-WORDPRESS-EXCLUSION-RULES.conf

```diff
@@ -162,6 +162,20 @@ SecAction \
 # [ Content editing ]
 #
 
+# Edit post
+# /wp-admin/post.php, /wp-admin/post.php?t=1475416064853
+# Themes do not properly escape post_title in HTML, so beware of XSS
```

Comment:
Yeah that may be true... In any case you wouldn't want `<script>` in the title :)